CHANGELOG.md

```diff
@@ -9,11 +9,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
-- The service certificate is now returned as part of the `/node/network/` endpoint response.
+- The service certificate is now returned as part of the `/node/network/` endpoint response (#2442).
+
+### Changed
+
+- `kv::Map` is now an alias to `kv::JsonSerialisedMap`, which means all existing applications using `kv::Map` will now require `DECLARE_JSON...` macros (instead of `MSGPACK_DEFINE`) for custom key and value types. Note that this change may affect throughput of existing applications (#2449).
```

Comment:
I think we should also mention that msgpack is no longer installed or visible in their include paths. So any `MSGPACK_DEFINE`s in app code, that were previously necessary, actually need to be _removed_ (and replaced with `DECLARE_JSON_...` if they didn't exist.

Also perhaps a suggestion to restore the lost throughput - "use `Blit` if you have simple types, or a custom serialiser"?